lib/oxidized/model/gcombnps.rb

```diff
@@ -0,0 +1,86 @@
+class GcomBNPS < Oxidized::Model
+  # For switches from GCOM Technologies Co.,Ltd. running the "Broadband Network Platform Software"
+  # Author: Frederik Kriewitz <frederik@kriewitz.eu>
+  #
+  # tested with:
+  #  - S5330 (aka Fiberstore S3800)
+
+  prompt /^\r?([\w.@()-]+?(\(1-16 chars\))?[#>:]\s?)$/ # also match SSH password promt (post_login commands are sent after the first prompt)
+  comment  '! '
+
+# alternative to handle the SSH login, but this breaks telnet
+#  expect /^Password\(1-16 chars\):/ do |data|
+#      send @node.auth[:password] + "\n"
+#      ''
+#  end
+
+  # handle pager (can't be disabled?)
+  expect /^\.\.\.\.press ENTER to next line, CTRL_C to quit, other key to next page\.\.\.\.$/ do |data, re|
+   send ' '
+   data.sub re, ''
+  end
+
+  cmd :all do |cfg|
+    cfg = cfg.gsub " \e[73D\e[K", '' # remove garbage remaining from the pager
+    cfg.each_line.to_a[1..-2].join
+  end
+
+  cmd :secret do |cfg|
+    cfg.gsub! /^(snmp-server community)\s+[^\s]+\s+(.*)/, '\\1 <community hidden> \\2'
+    cfg
+  end
+
+  cmd 'show running-config' do |cfg|
+    cfg
+  end
+
+  cmd 'show interface sfp' do |cfg|
+    cfg = cfg.each_line.to_a
+
+    cfg = cfg
+      .reject { |line| line.match /^  Temperature/ }
```

Comment:
A lot of the other models do it this way:

https://github.com/ytti/oxidized/blob/master/lib/oxidized/model/aireos.rb#L45

Can you switch to that or was it done like this for a reason?